app/Providers/AppServiceProvider.php

```diff
@@ -74,12 +75,16 @@ private function bootCustomBladeDirectives()
 
     private function configureMorphAliases()
     {
-        Relation::morphMap([
+        $sensor_types = [];
+        foreach (collect(Sensor::getTypes()) as $type) {
+            $sensor_types[ucwords($type)] = \App\Models\Sensor::class;
```

Comment:
The event_log code uses ucfirst().

I wonder if we should change the eventlog to be lowercase.  We could use translations to have nice display names so instead of `Power_factor` > `Power Factor`

The collect() call here is not needed.

Reply:
if i do it without ucwords, no sensor will be found. Reason is the niced sensor types in eventlog.related

collect is removed

Reply:
you had no problem removing the sensor_descr from the event_log call, but don't want to change the case there?